lib/srv/db/mysql/protocol/response.go

```diff
@@ -16,7 +16,9 @@ limitations under the License.
 
 package protocol
 
-import "github.com/gravitational/trace"
+import (
```

Comment:
Is this necessary?

Reply:
no, I guess my editor auto formatted it like that.